src/main/java/com/bina/varsim/util/VCFparser.java

```diff
@@ -661,15 +661,22 @@ public Variant parseLine() {
             return null;
         }
 
+        Variant variant = null;
+
         try {
-            return processLine(line);
+            variant = processLine(line);
         } catch (Exception e) {
             //TODO: right now just be lazy, die on any error
             log.fatal(e.getMessage() + "\n" + line);
             e.printStackTrace();
             System.exit(255);
         }
-        return null;
+
+        if (variant == null && !line.startsWith("#")) {
+            log.warn("Returned null variant for line " + line);
```

Comment:
Great logging message! I wonder if you can remove redundant logging messages in `processLine` that print the contents of a VCF record for null return value? Otherwise, I can do this in another PR. Thanks!